src/overlays/actors/ovl_player_actor/z_player.c

```diff
@@ -7,1096 +7,13991 @@
+void func_8084EAC0(Player* this, GlobalContext* globalCtx);
+void func_8084ECA4(Player* this, GlobalContext* globalCtx);
+void func_8084EED8(Player* this, GlobalContext* globalCtx);
+void func_8084EFC0(Player* this, GlobalContext* globalCtx);
+void func_8084F104(Player* this, GlobalContext* globalCtx);
+void func_8084F390(Player* this, GlobalContext* globalCtx);
+void func_8084F608(Player* this, GlobalContext* globalCtx);
+void func_8084F698(Player* this, GlobalContext* globalCtx);
+void func_8084F710(Player* this, GlobalContext* globalCtx);
+void func_8084F88C(Player* this, GlobalContext* globalCtx);
+void func_8084F9A0(Player* this, GlobalContext* globalCtx);
+void func_8084F9C0(Player* this, GlobalContext* globalCtx);
+void func_8084FA54(Player* this, GlobalContext* globalCtx);
+void func_8084FB10(Player* this, GlobalContext* globalCtx);
+void func_8084FBF4(Player* this, GlobalContext* globalCtx);
+s32 func_8084FCAC(Player* this, GlobalContext* globalCtx);
+void func_8084FF7C(Player* this);
+void func_8085002C(Player* this);
+s32 func_80850224(Player* this, GlobalContext* globalCtx);
+void func_808502D0(Player* this, GlobalContext* globalCtx);
+void func_808505DC(Player* this, GlobalContext* globalCtx);
+void func_8085063C(Player* this, GlobalContext* globalCtx);
+void func_8085076C(Player* this, GlobalContext* globalCtx);
+void func_808507F4(Player* this, GlobalContext* globalCtx);
+void func_80850AEC(Player* this, GlobalContext* globalCtx);
+void func_80850C68(Player* this, GlobalContext* globalCtx);
+void func_80850E84(Player* this, GlobalContext* globalCtx);
+void func_80851008(GlobalContext* globalCtx, Player* this, void* anim);
+void func_80851030(GlobalContext* globalCtx, Player* this, void* anim);
+void func_80851050(GlobalContext* globalCtx, Player* this, void* anim);
+void func_80851094(GlobalContext* globalCtx, Player* this, void* anim);
+void func_808510B4(GlobalContext* globalCtx, Player* this, void* anim);
+void func_808510D4(GlobalContext* globalCtx, Player* this, void* anim);
+void func_808510F4(GlobalContext* globalCtx, Player* this, void* anim);
+void func_80851114(GlobalContext* globalCtx, Player* this, void* anim);
+void func_80851134(GlobalContext* globalCtx, Player* this, void* anim);
+void func_80851154(GlobalContext* globalCtx, Player* this, void* anim);
+void func_80851174(GlobalContext* globalCtx, Player* this, void* anim);
+void func_80851194(GlobalContext* globalCtx, Player* this, void* anim);
+void func_808511B4(GlobalContext* globalCtx, Player* this, void* anim);
+void func_808511D4(GlobalContext* globalCtx, Player* this, void* anim);
+void func_808511FC(GlobalContext* globalCtx, Player* this, void* anim);
+void func_80851248(GlobalContext* globalCtx, Player* this, void* anim);
+void func_80851294(GlobalContext* globalCtx, Player* this, void* anim);
+void func_808512E0(GlobalContext* globalCtx, Player* this, void* arg2);
+void func_80851368(GlobalContext* globalCtx, Player* this, CsCmdActorAction* arg2);
```

Comment:
theres a ton of stuff in here that could be named, he mentioned in my stream the other day that he plans to revisit them at a later time. The only stuff that got documentation/names is external stuff

so yeah id say the best things to review are external stuff in the header/player lib stuff and just that overall styling guidelines are met

Reply:
Gotcha. 👍

Reply:
its alot to review, so skimming is encouraged Kappa
if theres any smaller stuff it will be caught in later documentation passes, but as long as multiple people have confirmed its good overall then we should be set